aws/data_source_aws_route.go

```diff
@@ -202,3 +204,16 @@ func getRoutes(table *ec2.RouteTable, d *schema.ResourceData) []*ec2.Route {
 	}
 	return routes
 }
+
+// Helper: Create an ID for a route
+func resourceAwsRouteID(d *schema.ResourceData, r *ec2.Route) string {
```

Comment:
Moved here temporarily. Will need to clean up when support for managed prefix list IDs added.